drivers/soundwire/stream.c

```diff
@@ -501,7 +501,7 @@ static int sdw_prep_deprep_slave_ports(struct sdw_bus *bus,
 	}
 
 	/* Inform slaves about ports prepared */
-	sdw_do_port_prep(s_rt, prep_ch, SDW_OPS_PORT_POST_PREP);
+	sdw_do_port_prep(s_rt, prep_ch, prep ? SDW_OPS_PORT_POST_PREP : SDW_OPS_PORT_POST_DEPREP);
```

Comment:
This is super interesting as this causes the failure?
rt711 does not even implement the port_prep callback, so this is just a NOP on this:
https://sof-ci.01.org/linuxpr/PR4201/build3272/devicetest/index.html?model=ADLP_RVP_SDW_IPC4ZPH&testcase=check-suspend-resume-with-playback

Reply:
Only two drivers handling SDW_OPS_PORT_*:
sound/soc/codecs/cs42l42-sdw.c
sound/soc/codecs/wsa881x.c